### PR TITLE
Add seeding scripts. Seed currencies

### PR DESCRIPTION
**Pull request by cogicgoran**

### CODE REVIEW

File: prisma/migrations/20230108225544_init_migration/migration.sql [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "currencyId" INTEGER NOT NULL,
+
+    CONSTRAINT "Article_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "Currency_code_key" ON "Currency"("code");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "Marketplace_address_key" ON "Marketplace"("address");
```
Comment (banecogic) [inline prisma/migrations/20230108225544_init_migration/migration.sql:56]: Not sure if this should be unique. 🤔 
There might be multiple marketplaces at the same address.

File: prisma/migrations/migration_lock.toml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Please do not edit this file manually
+# It should be added in your version-control system (i.e. Git)
+provider = "postgresql"
```
Comment (banecogic) [inline prisma/migrations/migration_lock.toml:3]: new line at the end

File: prisma/seed.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  } catch (error: unknown) {
+    console.error(JSON.stringify(error, null, 2));
+    console.error("Failed to seed currencies");
+  }
+  finally {
+    await prisma.$disconnect();
+  }
+}
+
+main();
```
Comment (banecogic) [inline prisma/seed.ts:43]: empty line at the end

File: prisma/schema.prisma [hunk 4]
```diff
@@ -2,53 +2,55 @@
...
 model Category {
-    id    Int    @id @default(autoincrement())
-    name  String @db.VarChar(30)
-    icon  String @db.VarChar(30)
-    color String @db.Char(7)
+    id       Int       @id @default(autoincrement())
+    name     String    @db.VarChar(30)
+    icon     String    @db.VarChar(30)
+    color    String    @db.Char(7)
+    articles Article[]
```
Comment (banecogic) [inline prisma/schema.prisma:17]: Why do we need an article array inside category? To fetch it per category and show on page?

Comment (cogicgoran) [reply prisma/schema.prisma:17]: I'll have to check in why, however when I entered property `Category` on entity `Article`, my IDE was flagged with error, that's why I added this line 17.
image

**Pull request merged**